src/app/api/fetch/health/processed-data/route.ts

```diff
@@ -0,0 +1,34 @@
+import { NextRequest, NextResponse } from "next/server";
+import { MongoClient, ObjectId } from "mongodb";
+import { HealthJSONData } from "@/types/types";
+import dayjs from "dayjs";
+import utc from "dayjs/plugin/utc";
+
+dayjs.extend(utc);
+
+export async function GET() {
+  const uri = process.env.MONGODB_URI as string;
+  const client = new MongoClient(uri);
+  try {
+    const database = client.db("HealthData");
+    const datacollection = database.collection("SampleHealthData");
+    const data = (await datacollection
+      .find({}, { projection: { processed_data: 1 } })
+      .toArray()) as unknown as HealthJSONData[];
+    return NextResponse.json({
+      status: 200,
+      statusText: "OK",
+      data: data,
+      time: dayjs(),
+    });
+  } catch (e) {
+    console.error(e);
```

Comment:
Also not specific to this file, but let's add more context to this console.error like "Database operation failed" or something like that